### ISSUE TITLE
WhoReacted should not include hidden reactions in its counts

### ISSUE THREAD

**Issue by FichteFoll** [jaimeadf/BetterDiscordPlugins#403]
In the current version of WhoReacted, when you activate the "Hide self" and "Hide bots" options, the +N counts will still include them. This is inconsistent because this doesn't happen when there are only ignored reactions on a message.

Only ignored reactions: ![2023-04-02_20-39-45](https://user-images.githubusercontent.com/931051/229372294-be112582-c167-4dc5-8ca6-31c353240fa4.png)

One ignored and two other reactions: ![2023-04-02_20-40-09](https://user-images.githubusercontent.com/931051/229372325-13077195-b0e5-48c0-82f0-a3f4fc769ae4.png)

Furthermore, I am never interested in this ignored reactions increasing the "+N" number because I specifically want to hide them entirely (which is why I implemented it this way in the first place in #4) and if I *really* wanted to know how many reactions there are in total, I could just look at the number on the left which will always be there.

In my option, it makes no sense to include ignored reactions in the +N counts in any situation.

**Comment by jaimeadf** [comment]
I must agree with you after using WhoReacted for the last two days. The "+N" is useless and just occupies precious space most of the time. However, I think that when the "+N" appears, the hidden reactions should be counted as it is impossible to know if the people who haven't been fetched meet the filter's criteria. I'm going to remove the "+N" when the maximum avatar limit hasn't been reached and maybe add an option to fully disable it. In the latest version, I've moved away from using discord components and have full control over everything that is rendered, which means there are no limits on what can be modified.

**Comment by FichteFoll** [comment]
> I think that when the "+N" appears, the hidden reactions should be counted as it is impossible to know if the people who haven't been fetched meet the filter's criteria

I presume this is because you only fetch as many user details as you need to reach the number of maximum avatars to be displayed? Could I interest you into adding an option to perform such fetches for as many as the configured maximum for WhoReacted to operate for a single reaction? In my situation, I have very rarely more than 12 users on a single reaction and most of the times one of them is going to be a bot that I'm very not interesed in.
